Cinema Room Manager/task/src/cinema/Room.java

```diff
@@ -112,8 +111,8 @@ public String getSeatsView() {
         return sb.toString();
     }
 
-    public void buyTicket() {
-        selectSeat();
+    public void buyTicket(int rowNum, int seatNum) {
+        selectSeat(rowNum, seatNum);
```

Comment:
Call `seat_sell(rowNum, seatNum)` directly here and remove validation code as I was mentioned above

Reply:
Done.